tests/integration/unit_test/test_unit_test_dotnet6.py

```diff
@@ -18,4 +18,7 @@ class UnitTest_dotnet6_cookiecutter_aws_sam_hello_step_functions_sample_app(Unit
     code_directories = ["tests/StockBuyer.Test", "tests/StockChecker.Test", "tests/StockSeller.Test"]
 
 
+class UnitTest_dotnet6_cookiecutter_aws_sam_quick_start_sqs(UnitTestBase.DotNetCoreUnitTestBase):
+    directory = "dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet"
+    code_directories = ["test/project.Tests.cs"]
```

Comment:
@ammoheim can you please correct this line of code to remove .cs?
from:
["test/project.Tests.cs"]
to:
["test/project.Tests"]